server/BraveAlerterConfigurator.js

```diff
@@ -234,7 +268,10 @@ class BraveAlerterConfigurator {
     switch (fromAlertState) {
       case CHATBOT_STATE.STARTED:
       case CHATBOT_STATE.WAITING_FOR_REPLY:
-        if (toAlertState === CHATBOT_STATE.WAITING_FOR_CATEGORY) {
+        // let other responders know that this session has ended by a reset
+        if (toAlertState === CHATBOT_STATE.RESET) {
```

Comment:
These changes could be tested in `test/unit/BraveAlerterConfiguratorTest/getReturnMessageToOtherResponderPhoneNumbersTest.js`